sdk/program/src/blake_3.rs

```diff
@@ -0,0 +1,172 @@
+use crate::sanitize::Sanitize;
```

Comment:
I diff'd this file against hash.rs and most of the comments come from that.

First thing is a top level comment  `//! The `blake3` module provides functions for creating Blake3 hashes.`

Also, any reason this file isn't `blake3.rs`?